apigw/src/shared/config.ts

```diff
@@ -136,12 +136,16 @@ const certificateNames = Object.keys(certificates) as ReadonlyArray<
 export const devLoginEnabled =
   env('DEV_LOGIN', parseBoolean) ?? ifNodeEnv(['local', 'test'], true) ?? false
 
-const eadCallbackUrl = process.env.EAD_SAML_CALLBACK_URL
+const adCallbackUrl = process.env.EAD_SAML_CALLBACK_URL
```

Comment:
Just naming conventions: noticed, that there are env variables starting with both EAD and AD.